### ISSUE TITLE
PollClient.info always produces NullPointerException

### ISSUE THREAD

**Issue by andlinger** [InterNetX/java-domainrobot-sdk#17]
Calling PollClient.info always results in a NullPointerException.

```
2020-08-21 21:26:17,236 SEVERE [a.t.w.b.c.DomainrobotPollJob] (executor-thread-2) Failed to poll message from domainrobot.: java.lang.NullPointerException
	at org.domainrobot.sdk.client.clients.PollClient.info(PollClient.java:52)
```

**Comment by Jabanto** [comment]
Hello @andlinger

The NPE is fixed. The NPE comes when there any poll messages to show. 
Please remenber that for better support, share the code that you used to find the NPE or another Issue, then when can 
work better and fast the issue :). 

Here the changes to realese: 
https://github.com/InterNetX/java-domainrobot-sdk/pull/18

Best regards

**Comment by Ephenodrom** [comment]
Hello @andlinger 
The new release is ready and this is fixed with version 0.2.7.

Regards
